yagocd/resources/version.py

```diff
@@ -31,7 +31,6 @@
 from yagocd.util import since
 
 
-@since('16.6.0')
 class VersionManager(BaseManager):
     """
     The Version API allows users to get the GoCD server version details.
```

Comment:
As no instance of this has a version field which is accessed in session.py, something like this seems necessary:
```suggestion

    @property
    def raw_data(self):
        if not hasattr(self, '_raw_data'):
            self._raw_data = self.get()
        return self._raw_data

    @property
    def version(self):
        return self.raw_data.version
```

Reply:
I'm afraid that although the changes I suggested were correct, the suggestion was bound to the wrong location in the code. Now they are essentially commented out. Sorry for the mess, it's the first time I use the feature.